Makefile

```diff
@@ -44,7 +42,9 @@ docker:
 generate: .tmp/protoc/bin/protoc
 	@go install google.golang.org/protobuf/cmd/protoc-gen-go@a709e31e5d12
 	@go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
+	@go install github.com/jhump/protoreflect/desc/sourceinfo/cmd/protoc-gen-gosrcinfo@v1.14.1
```

Comment:
What's the reasoning behind generating these src info files?

Reply:
IDK, but we do it in grpcui so I just figured parity and reconciling differences probably good.

Reply:
👍 makes sense